charts/tekton-apps/values.yaml

```diff
@@ -42,20 +42,17 @@ argocd:
   rootPath: "/"
 
 eventlistener:
```

Comment:
we cannot remove parameters, because they are used to create triggers. But I renamed to 'trigger'. Although, we have `extraOverlays` in apps definitions, and that section also has name `eventlistener` so left name as is, otherwise we will need to rename it in all apps/components across all repositories.